packages/dataviews/src/types.ts

```diff
@@ -231,6 +231,11 @@ export type Field< Item > = {
 	 */
 	elements?: Option[];
 
+	/**
+	 * The list of options to pick from when using the field as an array.
+	 */
+	suggestions?: Option[];
```

Comment:
Why not using the existing `elements`? I understand it's because of [this](https://github.com/WordPress/gutenberg/blob/fe63dd5b398853fdba9a2132419562f195522299/packages/dataviews/src/dataform-controls/index.tsx#L54). That was added when we hadn't field type definitions, and perhaps it's time to update that logic. For example, what if remove those lines and make the field type definitions themselves handled it?

Reply:
I agree! I pushed some changes in [7094494](https://github.com/WordPress/gutenberg/pull/71136/commits/709449472191c92f3d270f0f628d801c7db3673c) for this, but I think we can make that logic even more streamlined.